types/tag/index.d.ts

```diff
@@ -64,43 +93,18 @@ export interface SelectableProps extends HTMLAttributesWeak {
 }
 
 export class Selectable extends React.Component<SelectableProps, any> {}
-interface HTMLAttributesWeak extends React.HTMLAttributes<HTMLElement> {
-    onClick?: any;
-}
-
-export interface TagProps extends HTMLAttributesWeak {
-    /**
-     * 标签类名前缀,提供给二次开发者用
-     */
-    prefix?: string;
 
-    /**
-     * 标签的类型
-     */
-    type?: 'normal' | 'primary';
 
-    /**
-     * 标签的尺寸（large 尺寸为兼容表单场景 large = medium）
-     */
-    size?: 'small' | 'medium' | 'large';
+export interface TagGroupProps extends React.HTMLAttributes<HTMLElement> {
+    prefix?: string;
+    rtl?: boolean;
```

Comment:
Can we make a global interface for `rtl`? `Tag`, `ClosableTag`, and `TagGroup` all need and lots of other components do too.